recipes/qt/5.x.x/conanfile.py

```diff
@@ -0,0 +1,710 @@
+            if getattr(self.options, module):
+                _enablemodule(module)
+
+    def requirements(self):
+        if self.options.openssl:
+            self.requires("openssl/1.1.1i")
+        if self.options.with_pcre2:
+            self.requires("pcre2/10.35")
+
+        if self.options.with_glib:
+            self.requires("glib/2.67.1")
+        # if self.options.with_libiconv: # QTBUG-84708
+        #     self.requires("libiconv/1.16")# QTBUG-84708
+        if self.options.with_doubleconversion and not self.options.multiconfiguration:
+            self.requires("double-conversion/3.1.5")
+        if self.options.get_safe("with_freetype", False) and not self.options.multiconfiguration:
+            self.requires("freetype/2.10.4")
+        if self.options.get_safe("with_fontconfig", False):
+            self.requires("fontconfig/2.13.92")
+        if self.options.get_safe("with_icu", False):
+            self.requires("icu/68.2")
+        if self.options.get_safe("with_harfbuzz", False) and not self.options.multiconfiguration:
+            self.requires("harfbuzz/2.7.4")
+        if self.options.get_safe("with_libjpeg", False) and not self.options.multiconfiguration:
+            if self.options.with_libjpeg == "libjpeg-turbo":
+                self.requires("libjpeg-turbo/2.0.6")
+            else:
+                self.requires("libjpeg/9d")
+        if self.options.get_safe("with_libpng", False) and not self.options.multiconfiguration:
+            self.requires("libpng/1.6.37")
+        if self.options.with_sqlite3 and not self.options.multiconfiguration:
+            self.requires("sqlite3/3.33.0")
+            self.options["sqlite3"].enable_column_metadata = True
+        if self.options.get_safe("with_mysql", False):
+            self.requires("libmysqlclient/8.0.17")
+        if self.options.with_pq:
+            self.requires("libpq/12.2")
+        if self.options.with_odbc:
+            if self.settings.os != "Windows":
+                self.requires("odbc/2.3.7")
+        if self.options.get_safe("with_openal", False):
+            self.requires("openal/1.20.1")
+        if self.options.get_safe("with_libalsa", False):
+            self.requires("libalsa/1.2.4")
+        if self.options.gui and self.settings.os == "Linux":
+            self.requires("xorg/system")
+            if not tools.cross_building(self, skip_x64_x86=True):
+                self.requires("xkbcommon/1.0.3")
+        if self.options.get_safe("opengl", "no") != "no":
+            self.requires("opengl/system")
+        if self.options.with_zstd:
+            self.requires("zstd/1.4.8")
+        if self.options.qtwebengine and self.settings.os == "Linux":
+            self.requires("expat/2.2.10")
+            self.requires("opus/1.3.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        shutil.move("qt-everywhere-src-%s" % self.version, "qt5")
+
+        for p in self.conan_data["patches"][self.version]:
```

Comment:
In a future update of the recipe we probably want to replace this with our smarter default line for patches, but it would be a waste of time to change this now